app/javascript/controllers/heart_controller.js

```diff
@@ -21,12 +22,29 @@ export default class extends Controller {
       this.animation.clickUnfave()
       this.favorited = false
       this.unfaved()
+      this.unfavore()
     } else {
       this.animation.clickFave()
       this.favorited = true
       this.faved()
+      this.favore()
     }
   }
+  unfavore() {
```

Comment:
to use `favorite/unfavorite` as the verbs here.

Reply:
Ah, looks like HeartController is defining faved/unfaved as the verbs. We could probably change those to favorite/unfavorite to be more explicit.